### DISCUSSION TITLE
How to set the max price?

### DISCUSSION THREAD

**Discussion by lozbinks** [jef/streetmerchant#2264] [Help] [answered]
How do I set the max price for a ps5 on amazon?

**Comment by Enkendu** [comment, marked as answer]
You find the line in the dotenv file that says max price for the ps5. Line 57 and maybe 58 adding in a dollar amount you are wanting to set. 

 Initially the file is called dotenv-example you would copy it and remove the -example so that it just reads dotenv then modify the values for line 57 and 58. By default those values are left blank. 

So it might read something like 

MAX_PRICE_SERIES_SONYPS5C=500
MAX_PRICE_SERIES_SONYPS5DE=500 

This value assumes you are willing to spend 500 dollars on those items. I have no idea what a ps5 is going for currently. Good luck!
